### ISSUE TITLE
NFS not working in Kubernetes portable multinode setup

### ISSUE THREAD

**Issue by artak116** [kubernetes/kubernetes#30987]
Hello,

We use k8s (version 1.3.4) in portable multi-node cluster. We have an issue regarding NFS  configuration. We cannot mount NFS shares in Pods via PV and PVC. Troubleshooting issue we find the some solution. **nfs-common** library missing in Hyperkube image( https://github.com/kubernetes/kubernetes/blob/v1.3.4/cluster/images/hyperkube/Dockerfile ). 
After manually install **nfs-common** in running kubelet container 
```# docker exec -it CONTAINER_ID apt-get install nfs-common```
NFS share start working. 
Can you provide us the solution how to implement this issue.
Please fix in coming version or recommend other solution .
 If it is normal do this steps manually ?  

P.S

Is this problem related only nfs volume plugin or this is common all other plugins (iSCSI, GlusterFS, etc)

**Comment by luxas** [comment]
This has been fixed at HEAD and will work in `v1.4`
(And in `v1.4.0-alpha.3` if you're experimental)

Please reopen the issue in `kube-deploy` if there are more issues with docker-multinode

**Comment by n1koo** [comment]
Should it be deployed on the current images at the moment? 

```
~/kube-deploy/docker-multinode# docker run gcr.io/google_containers/hyperkube-amd64:v1.3.7 bash -c "dpkg --list|grep nfs"
```

Ends up empty and `kubelet` logs are full of mount errors indicating missing package too 

While I see its in https://github.com/kubernetes/kubernetes/blob/1.3.7/cluster/images/hyperkube/Dockerfile#L31

And the `v1.3.7` seems to be quite recently rebuilt

```
IMAGE               CREATED             CREATED BY                                      SIZE                COMMENT
24db5b90d9c0        4 days ago          /bin/sh -c ln -s /hyperkube /apiserver  && ln   80 B         
```
